.travis.yml

```diff
@@ -1,10 +1,7 @@
 language: go
 
 go:
-  - 1.8
-
-install:
-  - echo "pass"
+  - 1.9
```

Comment:
maybe also tip?

Reply:
it's relevant this repository doesn't contain any go